roles/manage_folder/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Check Mandatory Variables Are Defined
+  ansible.builtin.assert:
+    that:
+      - manage_folder_datacenter_name is defined
+      - manage_folder_folder_name is defined and ((manage_folder_folder_name | length) > 0)
+      - manage_folder_hostname is defined
+      - manage_folder_username is defined
+      - manage_folder_password is defined
+    quiet: true
+    fail_msg: Variable must be set when using this role.
+
+- name: Fail If Folder Path Is Absolute
+  ansible.builtin.fail:
+    msg: Folder name should not be absolute. It should be relative to /<datacenter>/<type>
+  when: manage_folder_folder_name[0] == '/'
+
+- name: Manage Full Folder Path
```

Comment:
This would be way nicer to have part of the module `community.vmware.vcenter_folder` IMHO.

Reply:
Agree - maybe we can open a ticket for that?